chapter4/astroml_chapter4_Hypothesis_testing.ipynb

```diff
@@ -0,0 +1,289 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ed196920",
+   "metadata": {},
+   "source": [
+    "# Hypothesis Testing\n",
+    "\n",
+    "In statistics, we often ask whether a given sample agrees with a given model or hypothesis. For example, suppose we have data $\\{x_i\\}$ assumed to be drawn from a Gaussian distribution, $\\mathcal{N}(\\mu, \\sigma)$. Here, $\\mathcal{N}(\\mu,\\sigma)$ is our *null hypothesis*, and we are trying to *reject it* to show that we measured a particular effect. \n",
+    "##### Example\n",
```

Comment:
Having two `Example` headings doesn't look too good in the TOC, instead, I'll pull them into one heading.
```suggestion
    "##### Examples\n",
    
    "**Example 1:**\n",
```